content/en/blog/_posts/2016-10-00-Dynamic-Provisioning-And-Storage-In-Kubernetes.md

```diff
@@ -130,7 +130,7 @@ This claim will result in an SSD-like Persistent Disk being automatically provis
 
 
 
-Dynamic Provisioning can be enabled for a cluster such that all claims are dynamically provisioned without a storage class annotation. This behavior is enabled by the cluster administrator by marking one StorageClass object as “default”. A StorageClass can be marked as default by adding the storageclass.beta.kubernetes.io/is-default-class annotation to it.
```

Comment:
Just find this is an old blog. Maybe this change doesn't apply for it.

You may want to check the below doc to see if there is anything needing to change:
https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/storage/persistent-volumes.md

Reply:
hm, the general setup of this article still is ok i think. It references K8S 1.6 though, so i am not really sure if changing this here is a good idea?

Reply:
cc @msau42 :)

Reply:
yeah we don't change old blog posts.